packages/plugins/Web3Profile/src/locales/en-US.json

```diff
@@ -13,7 +13,7 @@
     "Flow_wallet": "Flow Wallet",
     "wallet_status_button_change": "Change",
     "plugin_wallet_on_connect": "Connect Wallet",
-    "wallet_verify_has_bound": "The wallet has been bound. Please switch wallets.",
+    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another wallet.",
```

Comment:
```suggestion
    "wallet_verify_has_bound": "The wallet has already been bound to {currentPersona} Please switch to another.",
```

Reply:
This is the confirmed text, no need to change.